admin-dev/themes/new-theme/js/pages/product/components/categories/category-tree-selector.ts

```diff
@@ -213,7 +213,7 @@ export default class CategoryTreeSelector {
    * Used to recursively create items of the category tree
    */
   private generateCategoryTree(treeCategory: TreeCategory): HTMLElement {
-    const categoryNode = this.generateTreeElement(treeCategory);
+    const categoryNode = this.generateTreeElement(treeCategory) as HTMLElement;
```

Comment:
Either `this.generateTreeElement` type should be changed or add a generic to it, but don't cast it, this is wrong